docker-compose.yml

```diff
@@ -1,17 +1,26 @@
 # docker-compose.yml
-version: '3'
+version: "3"
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:2.0.1
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+
+      - DB_ENGINE=${DB_ENGINE}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_USER_PASSWORD=${DB_USER_PASSWORD}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
```

Comment:
It is not required to use dynamic envʼs injection, you could hardcode values here.

Reply:
Oh, I thought its just more comfortable to use on practice. Fixed